src/routers/components/newBook.jsx

```diff
@@ -1,56 +1,36 @@
 import { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import BookList from './aticles';
+import { addBook } from '../../redux/books/bookSlice';
 
 const AddNewBook = () => {
-  const [book, setBook] = useState([
-    {
-      id: Math.floor(Math.random() * (9999 - 1000)),
-      title: '',
-      author: '',
-    },
-    {
-      id: Math.floor(Math.random() * (9999 - 1000)),
-      title: '',
-      author: '',
-    },
-  ]);
+  const dispatch = useDispatch();
 
   const [title, setTitle] = useState('');
   const [author, setAuthor] = useState('');
 
-  const handleChangeTitle = (e) => {
-    setTitle(e.target.value);
-  };
-
-  const handleChangeAuthor = (e) => {
-    setAuthor(e.target.value);
-  };
-
   const handleSubmit = (e) => {
     e.preventDefault();
 
     const newBook = {
-      id: Math.floor(Math.random() * (9999 - 1000)),
       title,
       author,
+      category: '',
     };
-    setBook([...book, newBook]);
-  };
 
-  const delBook = (id) => {
-    setBook([
-      ...book.filter((book) => book.id !== id),
-    ]);
+    dispatch(addBook(newBook));
+    setTitle('');
+    setAuthor('');
   };
 
   return (
     <div>
       <form onSubmit={handleSubmit}>
-        <input type="text" placeholder="input the book title" onChange={handleChangeTitle} />
-        <input type="text" placeholder="input the book author" onChange={handleChangeAuthor} />
+        <input type="text" placeholder="input the book title" onChange={(e) => setTitle(e.target.value)} />
+        <input type="text" placeholder="input the book author" onChange={(e) => setAuthor(e.target.value)} />
         <button type="submit">Add Book</button>
       </form>
-      <BookList booksProps={book} setBook={setBook} delBook={delBook} />
+      <BookList />
     </div>
```

Comment:
![image](https://github.com/Zven94/Bookstore/assets/93039359/5b97ab5c-098e-422a-9307-c626d723ec7d)

- [ ] Great work so far :+1: but it would be nice if you fix this issue i have faced so when i write the name of the book and author and clicked the `Add Book` button twice then one time it showed the book but second time it didn't show the book :fearful: You can see that in the image above :arrow_up: You can see the error by following steps :arrow_down:

1. Write Book name and Author name
2. Click on **Add Book** twice
3. Don't clear Book name and Author name while clicking Add Book twice :+1: